runtime/compiler/control/OptionsPostRestore.cpp

```diff
@@ -154,12 +163,11 @@ J9::OptionsPostRestore::iterateOverExternalOptions()
 
          case J9::ExternalOptions::XcompilationThreads:
             {
-            char *compThreadsOption = J9::Options::_externalOptionStrings[J9::ExternalOptions::XcompilationThreads];
-            int32_t argIndex = FIND_AND_CONSUME_RESTORE_ARG(EXACT_MEMORY_MATCH, compThreadsOption, 0);
+            int32_t argIndex = FIND_AND_CONSUME_RESTORE_ARG(EXACT_MEMORY_MATCH, optString, 0);
```

Comment:
`EXACT_MEMORY_MATCH` does not look right here

Reply:
Same response as https://github.com/eclipse-openj9/openj9/pull/16769#discussion_r1120255959